source/_components/telegram_bot.markdown

```diff
@@ -8,7 +8,7 @@ comments: false
 sharing: true
 footer: true
 logo: telegram.png
-ha_category: Hub
+ha_category: Notifications
```

Comment:
The component is a hub because it's not only notification but also supports other platforms.